Source/Urho3D/CSharp/Managed/Scene/Node.cs

```diff
@@ -34,9 +38,68 @@ public partial class Node
             return (T)GetComponent(typeof(T).Name, recursive);
         }
 
+        /// <summary>
+        /// Get first occurance of a component type
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
         public T GetComponent<T>() where T : Component
         {
             return (T)GetComponent(typeof(T).Name);
         }
+
+        /// <summary>
+        /// get all components of a type
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <returns></returns>
+        public ComponentList GetComponentsbyType<T>(bool recursive = false) where T : Component
```

Comment:
`GetComponentsbyType` does not follow a naming convention. I will not comment on rest of naming convention violations, way too many of those ;)

Also generic convenience methods should follow names of original methods so we API between c++ and c# matches. In this case name should be `GetComponents<T>`.

Reply:
so glad you pointed this out... I will update these